### PR TITLE
Display 15 products per page, sorted by updatedAt desc && introduce t…

### PR DESCRIPTION
**Pull request by ithiame**
…he next / previous CTA

### CODE REVIEW

File: src/routes/(app)/admin[[hash=admin_hash]]/product/+page.svelte [hunk 1]
```diff
@@ -102,8 +104,26 @@
...
+		{/each}
+	</div>
+
+	<div class="flex flex-row mx-auto mt-4 gap-4">
+		<input type="hidden" value={next} name="skip" />
+		{#if Number($page.url.searchParams.get('skip'))}
+			<button
+				class="btn btn-blue"
+				type="submit"
+				on:click={() => (next = Math.max(0, next - PRODUCT_PAGINATION_LIMIT))}>&lt Previous</button
```
[the remark this answers is not in the document]
Comment (coyotte508) [reply src/routes/(app)/admin[[hash=admin_hash]]/product/+page.svelte:120]: note that you can use icons (we already have some icon libs installed) & emojis

**Review by coyotte508 — COMMENTED** [review]
you should also search only relevant pictures - and only one picture per product - @ithiame 

Currently pictures for all products are still loaded

File: src/lib/types/Product.ts [hunk 2]
```diff
@@ -95,6 +95,7 @@ export const MAX_DESCRIPTION_LIMIT = 10000;
 
 export const DEFAULT_MAX_QUANTITY_PER_ORDER = 10;
 export const POS_PRODUCT_PAGINATION = 10;
+export const PRODUCT_PAGINATION_LIMIT = 15;
```
Comment (coyotte508) [inline src/lib/types/Product.ts:98]:
```suggestion
export const PRODUCT_PAGINATION_LIMIT = 25;
```

as discussed in the issue

**Review by coyotte508 — APPROVED** [review]

**Comment by Tirodem** [comment]
Pagination performances are now OK. We'll introduce  filter later, I'll leave the main issue open.

**Pull request merged**